### ISSUE TITLE
Linked templates does not return errors

### ISSUE THREAD

**Issue by smndhm** [vue-a11y/eslint-plugin-vuejs-accessibility#215]
I just realized that linked template are not triggering errors with the plugin ! Am I right  or did I miss something in my configuration ? Those linked template should also return accessibility errors.

In my case this kind of configuration does not return any error :(

```
// Vue file
<script>....</script>
<template src="./template.html" >

// template.html file
<span>👩</span>
```

**Comment by kddnewton** [comment]
I don't think it actually can in this case. The function of the plugin is to lint vue files, but it in this case you're trying to lint an HTML file. I'd say you probably want another plugin for the `template.html` file.

**Comment by smndhm** [comment]
There is no way to consider them to be part of the Vue file ? Since they are linked from it.
And templates are not standard html files since they have Vue attributes or tags.
I'm surprised you are not considering it.

**Comment by kddnewton** [comment]
I’d be happy to merge it if you open a PR for it no problem.

**Comment by smndhm** [comment]
I did have a look at it and it seems not possible :
https://github.com/vuejs/eslint-plugin-vue/issues/490
https://github.com/vuejs/vue-eslint-parser/issues/28
